stdlib/LinearAlgebra/src/symmetric.jl

```diff
@@ -855,16 +861,25 @@ function ^(A::Hermitian{T}, p::Real) where T
             return Hermitian(retmat)
         end
     else
-        return (F.vectors * Diagonal((complex.(F.values).^p))) * F.vectors'
+        retmat = (F.vectors * Diagonal((complex.(F.values).^p))) * F.vectors'
+        if T <: Real
+            return Symmetric(retmat)
```

Comment:
Can this be `Hermitian(retmat)` instead? This should narrow the possible return types.

Reply:
Oh, I guess `retmat` is complex in any case, so this won't apply. In that case, does returning a `Symmetric` complex matrix provide an advantage over a `Matrix`?

Reply:
Indeed, `retmat` is not Hermitian. Complex symmetric matrices are of limited utility, I mainly did it to be consistent with the `sqrt` function, which was already applying the wrapper.

Reply:
I looked it up, and it turns out one solve linear systems with complex symmetric matrices via `sysv!`. The function does not accept the wrapper, though, it wants a raw matrix as input.